provider/resources.go

```diff
@@ -6023,12 +6027,20 @@ func setupComputedIDs(prov *tfbridge.ProviderInfo) {
 		return attr(state, "multiRegionClusterName"), nil
 	}
 
+	prov.Resources["aws_guardduty_member_detector_feature"].ComputeID = func(
+		ctx context.Context, state resource.PropertyMap,
+	) (resource.ID, error) {
+		return attrWithSeparator(state, ",", "detectorId", "accountId", "name"), nil
```

Comment:
This resource currently doesn't support import, so I'm not 100% sure this is correct. Based on the `Read` method it looks like these three are required

https://github.com/hashicorp/terraform-provider-aws/blob/658a64579f91e53276d2a71da00aa43b854d3de7/internal/service/guardduty/member_detector_feature.go?plain=1#L143

Reply:
Ah yes. The ID allocation is best-effort. Technically new GUID is a valid ID. We've been trying to match ID  to import IDs where possible.